modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/dump/Dump.java

```diff
@@ -317,6 +323,15 @@ private void advance() {
         };
     }
 
+    /**
+     * @param part Partition number.
+     * @param compressed If {@code true} then compressed partition file.
+     * @return Dump partition file name
+     */
+    public static String getDumpPartFileName(int part, boolean compressed) {
```

Comment:
methods in classes within "internal" packages don't use "get", "set", "is" prefixes.